transport/http/server.go

```diff
@@ -17,7 +17,7 @@ type Server struct {
 	before       []RequestFunc
 	after        []ServerResponseFunc
 	errorEncoder ErrorEncoder
-	finalizer    ServerFinalizerFunc
+	finalizer    []ServerFinalizerFunc
```

Comment:
Is line 87 still correct or we should check the `len` of the slice?

Reply:
it is correct as it is not initialized when not appended to, but since len is more explicit and handles both `nil` and `[]{}`I will change